src/Sql/dbo/Stored Procedures/OrganizationPasswordManager_Update.sql

```diff
@@ -0,0 +1,31 @@
+CREATE PROCEDURE [dbo].[OrganizationPasswordManager_Update]
+    @OrganizationId UNIQUEIDENTIFIER,
+    @Plan [nvarchar](50),
+    @PlanType [tinyint],
+    @Seats [int],
+    @MaxCollections [smallint],
+    @UseTotp [bit],
+    @UsersGetPremium [bit],
+    @Storage [bigint],
+    @MaxStorageGb [smallint],
+    @MaxAutoscaleSeats [int]
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    UPDATE
+        [dbo].[OrganizationPasswordManager]
+    SET
+        [Plan] = @Plan,
+        PlanType = @PlanType,
```

Comment:
I believe we should use `[]` around updates since that seems to be the convention in other stored procedures

Reply:
Thanks for pointing this out 👍